modules/placement-driver/README.md

```diff
@@ -0,0 +1,13 @@
+# Placement driver
+
+The module responses for location of leaseholder node in the replication group. The placement driver decides a node which can hold a lease 
+and approves with the replication group.
+
+## Leaseholder management
+Placement driver manager depends on Metastorage and a logical topology. The manager listens change of group members (due to a particular 
+data zone shrink or growing) and appear / disappear nodes in logical topology. Every time when a new node become available (appeared in the 
+topology and added to the group members) or unavailable (disappeared from the topology or removed to the group members) Placement driver 
+choosing a leaseholder taking into these modifications.
```

Comment:
> Every time when a new node become available (appeared in the 
topology and added to the group members) or unavailable (disappeared from the topology or removed to the group members) Placement driver 
choosing a leaseholder taking into these modifications.

What do you mean?